src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -137,7 +137,10 @@ def test_vm_show_list_sizes_list_ip_addresses(self, resource_group):
             self.check('[0].virtualMachine.resourceGroup', '{rg}'),
             self.check('length([0].virtualMachine.network.publicIpAddresses)', 1),
             self.check('[0].virtualMachine.network.publicIpAddresses[0].ipAllocationMethod', self.kwargs['allocation'].title()),
-            self.check('type([0].virtualMachine.network.publicIpAddresses[0].ipAddress)', 'string')
+            self.check('type([0].virtualMachine.network.publicIpAddresses[0].ipAddress)', 'string'),
```

Comment:
please check the real value, not type by creating a zonal vm

Reply:
Will do!